internal/controllers/machinedeployment/machinedeployment_status_test.go

```diff
@@ -538,17 +554,21 @@ func Test_setScalingUpCondition(t *testing.T) {
 				Status: metav1.ConditionFalse,
 				Reason: clusterv1.MachineDeploymentNotScalingUpReason,
 			},
+			expectedPhase: clusterv1.MachineDeploymentPhaseRunning,
```

Comment:
Maybe we should use phase deleting here, running is not ideal (maybe a follow-up)

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/12220